stbt/__init__.py

```diff
@@ -1357,6 +1358,7 @@ def as_precondition(message):
     try:
         yield
     except UITestFailure as e:
+        debug("Original exception:\n%s" % traceback.format_exc(e))
```

Comment:
Making the logging more explicit would probably be helpful. Something like:

    stbt.as_precondition caught a UITestFailure exception and will re-raise it as PreconditionError. The original exception was: